api/app/c_haines/severity_index.py

```diff
@@ -7,8 +7,11 @@
 import tempfile
 import logging
 import json
+try:
+    import ogr
```

Comment:
Do we need this? Isn't everything on a version of osgeo where we can just do a straight up `from osgeo import ogr`?

Reply:
I just do this because I'm assuming `import ogr` works on other machines without `from osgeo import ogr` working but I'll change it